vector-im/element-android
# [Session manager] Multi-session signout (PSG-857)

## Type of change


## Content


Adding capability to signout several sessions at once. On SDK side, adding new `delete_devices` endpoint.

## Motivation and context


Closes #7418 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/198031771-2a43b006-59bb-40cb-ad0e-2094efdc1201.gif" width=300 />

## Tests



- Enable the new session manager labs setting
- Go to Settings -> Security & Privacy -> Show all sessions
- In the other sessions section check you have the action to signout all other sessions in the overflow menu
- Press view All button (you need more than 5 other sessions)
- Check you have the signout all sessions action in the overflow menu
- Go into select mode
- Check there is the signout button outside the overflow menu when there is at least one selected session
- Select several sessions
- Press the signout button
- Confirm the action
- Check the selected session disappear from the list
- Check the select mode is left

## Tested devices

- OS version(s): Android 11

## Checklist
